README.md

```diff
@@ -1 +1,52 @@
-# gpc-scenario-numbering
\ No newline at end of file
+# gpc-scenario-numbering
+
+![Downloads](https://img.shields.io/npm/dw/gpc-scenario-numbering?style=flat-square) ![Version@npm](https://img.shields.io/npm/v/gpc-scenario-numbering?label=version%40npm&style=flat-square) ![Version@git](https://img.shields.io/github/package-json/v/gherking/gpc-scenario-numbering/master?label=version%40git&style=flat-square) ![CI](https://img.shields.io/github/workflow/status/gherking/gpc-scenario-numbering/CI/master?label=ci&style=flat-square) ![Docs](https://img.shields.io/github/workflow/status/gherking/gpc-scenario-numbering/Docs/master?label=docs&style=flat-square)
+
+The ScenarioNumbering precompiler is responsible for adding an index to all scenarios and scenario outlines.
+
+## Example
+
+```gherkin
+  @tag1
+  Scenario Outline: Test language (<language>)
```

Comment:
I'd suggest adding a scenario as well, to show it works with both scenario and scenario outline and how it adds index to them.